packages/main/src/MultiComboBox.js

```diff
@@ -420,13 +420,26 @@ class MultiComboBox extends UI5Element {
 		this.fireSelectionChange();
 	}
 
+	_onAfterClose() {
+		this._tokenizer.expanded = false;
```

Comment:
Changing properties of composed components imperatively should only be done as a last resort, ideally this should be via the `.hbs`